3rdPartySupport.md

```diff
@@ -14,11 +14,35 @@ If you know of any plugin/mod that fits into this list, please add it to
 [this config](https://github.com/BlueMap-Minecraft/BlueMapWiki/edit/master/assets/addon_browser/addons.conf) 
 and make a PR on GitHub! :)
 
-> **Important:**<br>
+> **Info:**<br>
 > Make sure the plugin/mod is compatible with your version of BlueMapAPI. You can see the API-Version of your BlueMap
 > on the [official download page](https://github.com/BlueMap-Minecraft/BlueMap/releases/latest). *("Targets: BlueMapAPI: v?.?.?")*
+{: .info }
+
+> **Important:**<br>
+> We **do not** exhaustively review every aspect of every entry on this list.     
+> We also **do not** guarantee the safety of your system when you install any addons from this list.  
+> We **do try** to remove actively harmful addons as soon as possible.  
+> See [below](#our-addon-policy) for more info!
 {: .info .important }
 
 ----
 
 <addon-browser></addon-browser>
+
+----
+
+## Our addon policy
+We allow every addon *(plugin, mod, resource pack, external tool)* on this list as long as it generally works and uses
+BlueMap's API or was at least partially made for it in some way.  
+That means we do not exhaustively test and review every aspect of every new entry, and we cannot guarantee that the addons on
+this list will work for your use case.   
+We also do not guarantee the safety of your system when you install any
+addons from this list, even though we try to remove actively harmful addons as soon as possible.
+
+**Reasons for a removal of an addon include:**
+- The addon being outdated and no longer maintained
+- Fatal issues/bugs that could irreversibly harm your server
+- Malware, Spyware and other intentionally harmful software
+
+If you feel like an addon on this list fulfills one of these criteria, report it (Github or Discord) and we will decide if the issue warrants a removal.
```

Comment:
I know this is mostly my own draft but I just noticed that we don't really mention that the reasons for removal are just examples and if there is an issue someone perceives as serious enough for a removal but it isn't explicitly mentioned, they should still just report it and we will decide on a case by case basis.  
Or maybe it isn't necessary and pretty obvious with some common sense, idk :D

Reply:
Good point, i changed the last sentence a bit :)